app/home/dashboard.njk

```diff
@@ -15,8 +15,27 @@
     {% for key, section in sections %}
       {% if canAccess(section.permission) %}
         <section class="govuk-grid-column-one-half govuk-!-margin-bottom-6">
-          <h2 class="govuk-heading-m govuk-!-margin-bottom-3">
+          <h2 class="govuk-heading-m govuk-!-margin-bottom-1">
             {{ t(section.heading) }}
+
+            <span class="govuk-!-display-block">
+            {% if section.period === 'week' %}
+              {{ dateRange | formatDateRange }}
+              {{ appTime({
+                datetime: dateRange | formatISOWeek,
+                text: '(' + t('actions::current_week') | lower + ')',
```

Comment:
I'd avoid lowercasing this I think. Then it's consistent with how it's being used in the side panel.